### PR TITLE
Update Cloud IDE ML tutorial

### PR DESCRIPTION
**Pull request by TJaniF**
Updating in anticipation of the webinar on Thursday, so this one is semi-urgent :)

### CODE REVIEW

**Review by kentdanas — CHANGES_REQUESTED** [review]
Looks good! Only requesting changes because I think creating an Astro account should still only be a prereq instead of a step, and we can update the prereq to say something like 'An Astro account. A free 14-day trial is available'

File: learn/cloud-ide-tutorial.md [hunk 1]
```diff
@@ -36,60 +36,63 @@ To get the most out of this tutorial, make sure you have an understanding of:
...
 - A Personal Access Token for your GitHub account. To create a personal access token, see the [official GitHub documentation](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token).
+- Optional. An account in one of the following database services, which are currently supported in the Astro Cloud IDE: [GCP BigQuery](https://cloud.google.com/bigquery/docs/quickstarts), [Postgres](https://www.postgresql.org/docs/current/tutorial-start.html), [Snowflake](https://docs.snowflake.com/en/user-guide-getting-started.html) or [AWS Redshift](https://docs.aws.amazon.com/redshift/latest/gsg/getting-started.html). Additionally you will need your login credentials to create the connection to your database. If you do not have a database account, you can still complete the main steps in this tutorial by using the in memory database and skipping [Step 3](#step-3-optional-configure-a-connection), as well as [Step 13](#step-13-optional-deploy-your-dag-to-astro) and [Step 14](#step-14-optional-run-your-dag-on-astro).
+
+## Step 0: Create an Astro account
```
[the remark this answers is not in the document]
Comment (TJaniF) [reply learn/cloud-ide-tutorial.md]: I think my braincell reasoned it is in between a step and a prereq, hence step 0, but yes can definitely move it back to the prereqs 😅

**Review by TJaniF — COMMENTED** [review]
Thank you for the review @kentdanas !

I realized we can make the prereq section way less daunting because all but the Astro account and Org/Workspace are only needed for the last 2 steps, let me know what you think :)

File: learn/cloud-ide-tutorial.md [hunk 2]
```diff
@@ -36,60 +36,64 @@ To get the most out of this tutorial, make sure you have an understanding of:
...
-- A Workspace in which you are either a Workspace Editor or a Workspace Admin.
-- An account in one of the following database services, which are currently supported in the Astro Cloud IDE: [GCP BigQuery](https://cloud.google.com/bigquery/docs/quickstarts), [Postgres](https://www.postgresql.org/docs/current/tutorial-start.html), [Snowflake](https://docs.snowflake.com/en/user-guide-getting-started.html) or [AWS Redshift](https://docs.aws.amazon.com/redshift/latest/gsg/getting-started.html). Additionally you will need your login credentials to create the connection to your database.
-- A GitHub account with access to a private or public repository that contains an Airflow Project created by the [Astro CLI](https://docs.astronomer.io/astro/cli/install-cli) 
+- An Astro account. If you do not already have an Astro account, [sign up for a free trial](https://www.astronomer.io/try-astro/) and follow the onboarding flow to create your first Organization and Workspace.
+
+To complete the optional steps [13](#step-13-optional-deploy-your-dag-to-astro) and [14](#step-14-optional-run-your-dag-on-astro) at the end of this tutorial, you will also need:
```
Comment (kentdanas) [inline learn/cloud-ide-tutorial.md:41]:
```suggestion
[Optional] To complete steps [13](#step-13-optional-deploy-your-dag-to-astro) and [14](#step-14-optional-run-your-dag-on-astro) at the end of this tutorial, you will also need:
```

**Review by kentdanas — APPROVED** [review]
Looks good!

**Comment by TJaniF** [comment]
Thank you Kenten!

FYI to @jwitz and @lzdanski Kenten and I decided to sneaky-merge this update past you because we will direct users to this tutorial in the webinar today and the old version is very outdated. This update is mostly changing button/field names that changed in the UI, all new screenshots and rewording the requirements to show that you can do most of the tutorial with just a trial Astro account without needing to worry about GH or a database. :)

**Pull request merged**